_docs/tasks/traffic-management/circuit-breaking.md

```diff
@@ -0,0 +1,237 @@
+    
+    HTTP/1.1 200 OK
+    server: envoy
+    date: Tue, 16 Jan 2018 23:47:00 GMT
+    content-type: application/json
+    access-control-allow-origin: *
+    access-control-allow-credentials: true
+    content-length: 445
+    x-envoy-upstream-service-time: 36
+    
+    {
+      "args": {}, 
+      "headers": {
+        "Content-Length": "0", 
+        "Host": "httpbin:8000", 
+        "User-Agent": "istio/fortio-0.6.2", 
+        "X-B3-Sampled": "1", 
+        "X-B3-Spanid": "824fbd828d809bf4", 
+        "X-B3-Traceid": "824fbd828d809bf4", 
+        "X-Ot-Span-Context": "824fbd828d809bf4;824fbd828d809bf4;0000000000000000", 
+        "X-Request-Id": "1ad2de20-806e-9622-949a-bd1d9735a3f4"
+      }, 
+      "origin": "127.0.0.1", 
+      "url": "http://httpbin:8000/get"
+    }
+    ```    
+    
+You can see the request succeeded! Now, let's break something.    
+    
+### Tripping the circuit breaker:
+
+In the circuit-breaking settings, we specified `maxConnections: 1` and `httpMaxPendingRequests: 1`. This should mean that if we exceed more than one connection and request concurrently, we should see the istio-proxy open the circuit for further requests/connections. Let's try with two concurrent connections (`-c 2`) and send 20 requests (`-n 20`)
+
+    ```bash
+    kubectl exec -it $FORTIO_POD  -c fortio /usr/local/bin/fortio -- load -c 2 -qps 0 -n 20 http://httpbin:8000/get
+    
+    Fortio 0.6.2 running at 0 queries per second, 2->2 procs, for 5s: http://httpbin:8000/get
+    23:51:10 I httprunner.go:72> Starting http test for http://httpbin:8000/get with 2 threads at -1.0 qps
+    23:51:10 I periodic.go:153> Negative qps -1.000000 means max speed mode/no wait between calls
+    23:51:10 I periodic.go:186> WATCHER 1 First outstanding run starting, catching signal
+    Starting at max qps with 2 thread(s) [gomax 2] for exactly 20 calls (10 per thread + 0)
+    23:51:10 W http.go:617> Parsed non ok code 503 (HTTP/1.1 503)
+    23:51:10 I periodic.go:473> T000 ended after 98.10645ms : 10 calls. qps=101.93009735853249
+    23:51:10 I periodic.go:473> T001 ended after 106.418465ms : 10 calls. qps=93.96865478185576
+    Ended after 106.474079ms : 20 calls. qps=187.84
+    Aggregated Function Time : count 20 avg 0.010215375 +/- 0.003604 min 0.005172024 max 0.019434859 sum 0.204307492
+    # range, mid point, percentile, count
+    >= 0.00517202 <= 0.006 , 0.00558601 , 5.00, 1
+    > 0.006 <= 0.007 , 0.0065 , 20.00, 3
+    > 0.007 <= 0.008 , 0.0075 , 30.00, 2
+    > 0.008 <= 0.009 , 0.0085 , 40.00, 2
+    > 0.009 <= 0.01 , 0.0095 , 60.00, 4
+    > 0.01 <= 0.011 , 0.0105 , 70.00, 2
+    > 0.011 <= 0.012 , 0.0115 , 75.00, 1
+    > 0.012 <= 0.014 , 0.013 , 90.00, 3
+    > 0.016 <= 0.018 , 0.017 , 95.00, 1
+    > 0.018 <= 0.0194349 , 0.0187174 , 100.00, 1
+    # target 50% 0.0095
+    # target 75% 0.012
+    # target 99% 0.0191479
+    # target 99.9% 0.0194062
+    Code 200 : 19 (95.0 %)
+    Code 503 : 1 (5.0 %)
```

Comment:
I guess this is the interesting part of the output, 5% of 503s

you could maybe call fortio with `-loglevel warning` to reduce the flood of output

Reply:
I think that's a useful thing -- would love to see how much gets trimmed -- but i don't see any effect when i change the log level. may take a look at https://github.com/istio/fortio/issues/85 ?

Reply:
loglevel only trims logger output (lines with timestamp an level letter `I` `W`...)
I guess you can just delete the lines that feel noisy and put [...]